lib/horizontal-repositories.js

```diff
@@ -13,6 +13,7 @@ class HorizontalLabels {
   constructor() {
     this._repositories = []; // initialize before the fetch
     this.labels = fetch(HR_LABELS_URL).then(data => data.json())
+      .then(labels => labels.filter(l => l.repo))
```

Comment:
That change is ok but once the pull request in w3c.github.io gets merged, I'll ned to update the location of the labels as well since hr-labels.json will disappear.

Reply:
Doing that shouldn't be an emergency since I made hr-labels.json a symlink to common-labels.json, so https://w3c.github.io/hr-labels.json still works.